### ISSUE TITLE
buildtools.jar being called incorrectly 

### ISSUE THREAD

**Issue by MarkehMe** [benediktschmitt/emsm#71]
The buildtools are not being called under the user it should be, and it should be passing at least 1g of memory or it can fail.  

I had to [modify this](https://github.com/benediktschmitt/emsm/blob/fbd2b3a1e03f34913be989f4dfe49bc4b4fe2bd2/emsm/core/server.py#L963) and change it to:

```python
...

  with subprocess.Popen(
      ["java", "-Xmx1g", "-jar", buildtools],
      cwd = build_dir,
      stdout = subprocess.PIPE,
      stderr = subprocess.PIPE
  ) as proc:

...
```

And execute the command like so:
```
sudo su minecraft -c "minecraft -h"
```

**Comment by benediktschmitt** [comment]
Thank you! I'll fix this one together with #68.

**Comment by benediktschmitt** [comment]
May be I'm wrong, but doesn't set *Xmx1g" the maximum heap size to 1gb. Do you mean "Xms1g"?
